tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -571,44 +575,44 @@ tf_cc_test(
     ],
 )
 
-cc_library(
-    name = "cusolver_context",
-    srcs = ["cusolver_context.cc"],
-    hdrs = ["cusolver_context.h"],
-    deps = [
-        "//tensorflow/compiler/xla:statusor",
-        "//tensorflow/compiler/xla:types",
-        "//tensorflow/compiler/xla:util",
-        "//tensorflow/core:lib",
-        "//tensorflow/core:stream_executor_no_cuda",
-        "//tensorflow/stream_executor:blas",
-    ]
-    + if_cuda_is_configured(if_cuda([
-        "@local_config_cuda//cuda:cuda_headers",
-        "@local_config_cuda//cuda:cusolver",
-    ])),
-)
+#cc_library(
```

Comment:
do we need to comment out this target? perhaps use `if_cuda_is_configured()` to make `srcs` and `hdrs` empty?